internal/pkg/mount/pseudo.go

```diff
@@ -29,5 +33,10 @@ func PseudoSubMountPoints() (mountpoints *Points, err error) {
 	pseudo.Set("hugetlb", NewMountPoint("hugetlbfs", "/dev/hugepages", "hugetlbfs", 0, ""))
 	pseudo.Set("securityfs", NewMountPoint("securityfs", "/sys/kernel/security", "securityfs", unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_NODEV|unix.MS_RELATIME, ""))
 
+	if _, err := os.Stat(constants.EFIVarsMountPoint); err == nil {
```

Comment:
mount as `readonly`, remount in the code which writes the vars? better for security

Reply:
TODO: for later